lib/rules/indent.js

```diff
@@ -1029,8 +1204,13 @@ module.exports = {
 
             WhileStatement: node => addBlocklessNodeIndent(node.body, node),
 
+            "*": setCurrentUnknownNode,
+
+            "*:exit": saveOutermostUnknownNode,
+
             "Program:exit"() {
                 addParensIndent(sourceCode.ast.tokens);
+                ignoreUnknownNodes();
```

Comment:
We could do it this way. We have to find a way to map line number to unknown nodes and ignore them that way. Not sure how nested known nodes would work, since they would also have an incorrect desired indent until we set the correct dependency. I'll look into this after I fix the above issues.